test/GovPool.test.js

```diff
@@ -213,98 +324,36 @@ describe("GovPool", () => {
     await govPool.execute(proposalId);
   }
 
+  const assertBalanceDistribution = (balances, coefficients) => {
+    if (balances.length !== coefficients.length) {
+      assert.isTrue(false, "lengths mismatch");
+    }
```

Comment:
Let's remove this assertion as it tests the tests

Reply:
done